lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -443,7 +443,9 @@ export function detectTestingLibraryUtils<
      *
      * Not to be confused with {@link isUserEventMethod}
      */
-    const isUserEventUtil = (node: TSESTree.Identifier): boolean => {
+    const isUserEventUtil = (node: TSESTree.Identifier | null): boolean => {
+      if (!node) return false;
```

Comment:
Add brackets around body, to make it less error-prone

```suggestion
      if (!node) {
        return false;
      }
```